HW_1/task_2.py

```diff
@@ -0,0 +1,17 @@
+num = int(input("Введите целое число (от 0 до 100000): "))
+
+# Проверяем ограничения на ввод
+if num < 0 or num > 100000:
+    print("Некорректный ввод")
+else:
+    # Проверяем является ли число простым
+    if num > 1:
+        for i in range(2, int(num/2)+1):
```

Comment:
Можно использовать просто оператор целочисленного деления
```suggestion
        for i in range(2, num // 2 + 1):
```

Reply:
Но понизить количество итераций можно чуть сильнее